test/runtests.jl

```diff
@@ -14,15 +14,15 @@ push!(solvers, ECOSSolver(verbose=0))
 push!(solvers, GLPKSolverMIP())
 push!(solvers, SCSSolver(verbose=0, eps=1e-6))
 
-if isinstalled("Gurobi")
-    using Gurobi
-    push!(solvers, GurobiSolver(OutputFlag=0))
-end
-
-if isinstalled("Mosek")
-    using Mosek
-    push!(solvers, MosekSolver(LOG=0))
-end
+#if isinstalled("Gurobi")
```

Comment:
Why the commented block of code?

Reply:
We don't currently install Mosek and Gurobi for the tests, so these tests were only triggered if the user happened to have the packages installed. I commented them out because I didn't know whether we should remove the code or actually require and test those solvers along with the rest.

Reply:
We can't test the Gurobi (and I assume Mosek) without a CI server with liscenses.